src/main/webapp/file.xhtml

```diff
@@ -309,11 +309,11 @@
                                                     #{bundle['metrics.file.title']}
                                                     <ui:fragment rendered="#{!settingsWrapper.makeDataCountDisplayEnabled}">
                                                         <span class="glyphicon glyphicon-question-sign tooltip-icon" data-toggle="tooltip" data-placement="auto top" 
-                                                            data-trigger="hover" data-original-title="#{bundle['metrics.dataset.tip.default']}"></span>
+                                                            data-trigger="hover" data-original-title="#{bundle['metrics.file.tip.default']}"></span>
                                                     </ui:fragment>
                                                     <ui:fragment rendered="#{settingsWrapper.makeDataCountDisplayEnabled}">
                                                         <a tabindex="0" role="button" class="glyphicon glyphicon-question-sign tooltip-icon" data-toggle="popover" data-placement="auto top" 
-                                                            data-trigger="focus" data-html="true" data-content="#{bundle['metrics.dataset.tip.makedatacount']}"></a>
+                                                            data-trigger="focus" data-html="true" data-content="#{bundle['metrics.file.tip.makedatacount']}"></a>
```

Comment:
And this...

metrics.dataset.tip.makedatacount=Metrics collected using <a href="https://makedatacount.org/counter-code-of-practice-for-research-data/" target="_blank" rel="noopener"/>Make Data Count</a> standards.

... becomes...

metrics.file.tip.makedatacount=Individual file downloads are tracked in Dataverse but are not reported as part of the Make Data Count standard.